docs/ops.html

```diff
@@ -3570,7 +3572,7 @@ <h4 class="anchor-heading"><a id="others_monitoring" class="anchor-link"></a><a
   <h3 class="anchor-heading"><a id="zk" class="anchor-link"></a><a href="#zk">6.9 ZooKeeper</a></h3>
 
   <h4 class="anchor-heading"><a id="zkversion" class="anchor-link"></a><a href="#zkversion">Stable version</a></h4>
-  The current stable branch is 3.5. Kafka is regularly updated to include the latest release in the 3.5 series.
+  The current stable branch is 3.8. Kafka is regularly updated to include the latest release in the 3.8 series.
```

Comment:
This is still hardcoded; can we programmatically render the version? Or is it automatically updated by the release script?

Reply:
This is the ZooKeeper version. We don't have a variable to track that. At this point, it's likely it won't change anymore before we remove ZooKeeper, hence why I decided to just update the value.